index.html

```diff
@@ -77,6 +82,8 @@
                 rows="10"
                 cols="100"
                 required
+                spellcheck="false"
```

Comment:
Nice! Related issue at eutiveumsonho in case you want to dig in: 
- https://github.com/eutiveumsonho/eutiveumsonho/issues/77